src/algorithms/diamondsquare.jl

```diff
@@ -39,29 +39,33 @@ DiamondSquare() = DiamondSquare(0.5)
 """
     MidpointDisplacement()
 
-    Creates a midpoint-displacement algorithm object `MidpointDisplacement`. The degree of spatial autocorrelation is controlled by a parameter `H`,
-    which varies from 0.0 (low autocorrelation) to 1.0 (high autocorrelation) --- note this is non-inclusive and H = 0 and H = 1 will not behavive as expected.
+Creates a midpoint-displacement algorithm object `MidpointDisplacement`. The
+degree of spatial autocorrelation is controlled by a parameter `H`, which varies
+from 0.0 (low autocorrelation) to 1.0 (high autocorrelation) --- note this is
+non-inclusive and H = 0 and H = 1 will not behave as expected.
 
-    A similar algorithm, diamond-square, functions almost
-    identically, except that in diamond-square, the square step interpolates
-    edge midpoints from the nearest two corners and the square's center, where as 
-    `MidpointDisplacement` only intepolates from the nearest corners (see `DiamondSquare`).
+A similar algorithm, diamond-square, functions almost identically, except that
+in diamond-square, the square step interpolates edge midpoints from the nearest
+two corners and the square's center, where as `MidpointDisplacement` only
+intepolates from the nearest corners (see `DiamondSquare`).
 """
 struct MidpointDisplacement <: NeutralLandscapeMaker
-    H::Float64
+    H::Real
```

Comment:
If we want to use abstract types in structs, maybe make them parametric to make sure methods are inferred?
```julia
struct MidpointDisplacement{R<:Real} <: NeutralLandscapeMaker
    H::R
```